src/main/java/org/terasology/inGameHelp/ItemsCategoryInGameHelpRegistry.java

```diff
@@ -18,10 +18,24 @@
 import org.terasology.entitySystem.prefab.Prefab;
 import org.terasology.inGameHelp.components.HelpItem;
 
+/**
+ * Interface for dealing with the prefabs for the items
+ */
 public interface ItemsCategoryInGameHelpRegistry {
+
+    /**
+     * Add a Prefab and a variable amount of HelpItems
+     */
     void addKnownPrefab(Prefab prefab, HelpItem... helpItems);
```

Comment:
this associates HelpItems with prefabs, i.e., one prefab can be connected with many help items.